TimeWarpEngineering/timewarp-architecture
# Cramer/2023 08 27/Pages

- Move PageComponent into Stacked (We will add another Sidebar later)
- Rename PageComponent to StackedPage
- rename SiteFooterContent to CustomContent to indicate there is a default
- Add Sidebar page component
- Move Processing into its own state
- Rename Services to ServicesPage
- Rename Index to HomePage
- Add Processing to Global usings
- Use ProcessingState in footer
- update style a bit to add ring
- Logo now is just the image with no wrapper div
- Cleanup optional tags a bit
- Use ProcessingState
- Add quick readme
- Update to Blazor-State 10.0.0
- formatting
- Add SidebarState and open and close actions
- formatting
- Update Sidebar.  Mostly working.
- move to sm break point from md
- size the hamburger button
- Add class="h-5 w-5" to all icons in sidebar
- Update to latest heroicons :)
- Update Console output to be more meaningful.  Should I use Log?
- Move Profile Button
- Sidebar mostly working except transitions.
- Profile working
- Working on closing menus when anything else is clicked or user scrolls... NotifyLossOfInterest
- Update the TypeScript to include types from Blazor-State
- using typescript ts instead of js for the lib.module
- Update comments in tests
- Use new js location for dispatching action Spa.Counter.DispatchIncrementCountAction()
- NotifyLossOfInterest fires on scroll and click outside but isn't removing eventlistners.
- Starting on Transition Component to be used with Tailwind https://github.com/
- More Transtion progress
- Update coding standards
- Update Transition.js
- Update again
- WIP
- Update Azure.Identity Package as it had security vulnurability.
